tests/all/integration/api/helpers/test_scheduled_jobs.py

```diff
@@ -77,3 +83,30 @@ def test_delete_ticket_holder_with_valid_order_id(self):
             delete_ticket_holders_no_order_id()
             ticket_holder = TicketHolder.query.get(attendee_id)
             self.assertIsNot(ticket_holder, None)
+
+    def test_send_monthly_invoice(self):
+        """Method to test marking of event invoices as due"""
+
+        with self.app.test_request_context():
+            # ticket fee factory
+            ticket_fee_test = TicketFeesFactory()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'ticket_fee_test' is assigned to but never used](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4928052)